client/src/panels/panel_declarations/finances/historical_auth_exp/auth_exp_prog_spending.js

```diff
@@ -90,31 +94,19 @@ class AuthExpProgSpending extends React.Component {
     
     let graph_content;
     if(window.is_a11y_mode){
-      const historical_data = _.map(
-        exp, 
-        (exp_value,year_index) => ({
-          label: history_ticks[year_index],
-          data: [
-            formatter("compact1_written", exp_value, {raw: true}),
-            formatter("compact1_written", auth[year_index], {raw: true}),
-            null,
-          ],
-        })
-      );
-      
-      const planning_data = _.map(
-        progSpending, 
-        (progSpending_value, year_index) => ({
-          label: plan_ticks[year_index],
-          data: [
-            null,
-            null,
-            formatter("compact1_written", progSpending_value, {raw: true}),
-          ],
-        })
-      );
-  
-      const data = _.concat(historical_data, planning_data);
+      const all_ticks = _.chain(auth_ticks)
+        .concat(exp_ticks, plan_ticks)
+        .uniq()
+        .value();
+
+      const data = _.map(all_ticks, tick => ({
+        label: tick,
+        data: [
+          auth[_.findIndex(auth_ticks, d=>d===tick)] || null,
+          exp[_.findIndex(exp_ticks, d=>d===tick)] || null,
+          progSpending[_.findIndex(plan_ticks, d=>d===tick)] || null,
```

Comment:
Any cases where we'd want to preserve 0's? A null would be a gap in the graph, right?

Reply:
Hmmmm, maybe. I'll alter this to display a zero if there is one.